### ISSUE TITLE
Keep // comments

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#108]
Is there a reason why `/* */` comments within code are kept and `//` comments aren't?

**Comment by drkameleon** [comment]
Nice that you mention it, since someone also noted that (in an e-mail sent to me). And the whole issue is somewhat related to: https://github.com/drkameleon/iswift/issues/94

Basically:

- `/* *` are recognized (and converted), when they are top-level statements
- `//` are ignored

My intention currently is to make iSwift recognize `//` full-line (meaning: a line beginning with `//`).

And the rest of it (comments being embedded within the statement tree) may follow.

**Comment by jrmgx** [comment]
+1

Any comments (of any type) should stay after the translation :)

**Comment by JanX2** [comment]
+1

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0. 

----

NOTE: Both multi-line and single-line comments will be converted in all cases. With one catch: the **comments must not "break" any legal statement** in the middle (this will throw an error). e.g.

```objective-c
for (int i=0; i<10; i++) // this won't work
{
}

for (int i=0; i<10; i++) { // this will work
}
```
